frameworks.md

```diff
@@ -21,7 +21,7 @@ Test particularities:
 Suite particularities:
   * the start and end of a suite, even the global one, are emitted only if the suite contains at least a test or a child suite (i.e nested suites) that contains a test
 
-One interesting aspect is the execution of nested suite, which is done from the outer suite (i.e top level) to the inner most suite.
+One interesting aspect is the execution of nested suites, which is done quite in a source order. Practically, Mocha executes only suites in source order, when a suite is encountered, its start is emitted and then all its tests are executed, then the other suites.
```

Comment:
"is done quite in a source order" sounds pretty weird. I'm not sure what you're trying to express with that.

> "Mocha executes only suites in source order"

There's something about tests being run before siblings suites, right? You should make that explicit here.